test/smoke/testApps/TelemetryProcessors/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/SpringBootAutoTest.java

```diff
@@ -26,6 +27,13 @@ public void doMostBasicTest() throws Exception {
         assertEquals("/TelemetryProcessors/test", rd.getProperties().get("httpPath"));
         assertEquals(4, rd.getProperties().size());
         assertTrue(rd.getSuccess());
+
+        List<MessageData> logs = mockedIngestion.getAllMessageData();
```

Comment:
can we add comment for testing log processor?

Reply:
makes sense, can move the `logger.info` to the controller method so that it happens during the request, instead of during startup